libs/platform/src/lib/components/menu/menu-trigger.directive.ts

```diff
@@ -104,36 +106,34 @@ export class MenuTriggerDirective implements OnDestroy, AfterContentInit {
 
     /** @hidden Handled keypress which focus is on trigger element. */
     @HostListener('keydown', ['$event']) onTriggerKeydown($event: KeyboardEvent): void {
```

Comment:
Somewhere it's named `event` and somewhere else `$event`, as far as I know, we're not using `$` in such cases. Should it be unified to the `event`?

Reply:
made it uniform with other components. using `event`, removed `$event`.